### DISCUSSION TITLE
[HA] How can I install the newest version 24.1 on Home Assistant?

### DISCUSSION THREAD

**Discussion by szczukot** [john30/ebusd#1375] [Q&A]
Hi. At the moment I use HA Addon for install ebusd. But there is very old version of ebusd (23.2).
How can I install the newest version?

**Comment by hdehaseleer** [comment]
Hi, I've the same question.
In HA, the add-on is configured for "auto update new version".  
Even after two reboots of my RPI, the add-on says I still have version 23.2.6.  
And the add-on in HA is still saying "update available"
![{DF2753B4-AFAC-4F20-BE0D-5E53A163F82A}](https://github.com/user-attachments/assets/c36fb34a-415b-4d90-b1c5-b9ce1ff8bc0b)

**Comment by Honzik1234** [comment]
the same

**Comment by gerd12po** [comment]
the same :-(

**Comment by Pantalooney** [comment]
Same here! :-/

**Comment by verboEse** [comment]
see https://github.com/LukasGrebe/ha-addons/issues/151 ...

**Comment by szczukot** [comment]
https://github.com/LukasGrebe/ha-addons/pull/153
